config/resources/domain.json

```diff
@@ -6,9 +6,146 @@
+      "class": "sh:Shape",
+      "relationships": {
+        "targetClass": {
+          "predicate": "sh:targetClass",
+          "target": "resource",
+          "cardinality": "one"
+        },
+        "targetNode": {
+          "predicate": "sh:targetNode",
+          "target": "resource",
+          "cardinality": "one"
+        },
+        "targetHasConcept": {
+          "predicate": "ext:targetHasConcept",
+          "target": "concept",
+          "cardinality": "one"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/Shapes/"
+    },
+    "property-shape": {
+      "name": "propertyShape",
+      "class": "sh:PropertyShape",
+      "super": ["shape"],
+      "relationships": {
+        "path": {
+          "predicate": "sh:path",
+          "target": "resource",
+          "cardinality": "one"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/PropertyShapes/"
+    },
+    "node-shape": {
+      "name": "nodeShape",
+      "class": "sh:NodeShape",
+      "super": ["shape"],
+      "new-resource-base": "http://data.lblod.info/NodeShapes/"
+    },
+    "concept":{
+      "name": "concept",
+      "class": "ext:Concept",
+      "super": ["resource"],
+      "relationships": {
+        "template": {
+          "predicate": "ext:template",
+          "target": "template",
+          "cardinality": "many"
+        },
+        "relation": {
+          "predicate": "ext:relation",
+          "target": "relation",
+          "cardinality": "many"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/Concepts/"
+    },
+    "template": {
+      "name": "template",
+      "class": "ext:Template",
+      "attributes": {
+        "value": {
+          "type": "string",
+          "predicate": "ext:value"
+        }
+      },
+      "relationships": {
+        "mapping": {
+          "predicate": "ext:mapping",
+          "target": "mapping",
+          "cardinality": "many"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/Templates/"
+    },
+    "mapping": {
+      "name": "mapping",
+      "class": "ext:Mapping",
+      "attributes": {
+        "variable": {
+          "type": "string",
+          "predicate": "ext:variable"
+        }
+      },
+      "relationships": {
+        "expects": {
+          "predicate": "ext:expects",
+          "target": "shape",
+          "cardinality": "one"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/Mappings/"
+    },
+    "relation": {
+      "name": "relation",
+      "class": "ext:Relation",
+      "attributes": {
+        "expectedNumber": {
+          "type": "integer",
+          "predicate": "ext:expectedNumber"
+        },
+        "reason": {
+          "type": "string",
+          "predicate": "ext:reason"
+        }
+      },
+      "relationships": {
+        "concept": {
+          "predicate": "ext:concept",
+          "target": "concept",
+          "cardinality": "one"
```

Comment:
Should it be `"many"` ?

Reply:
ah yea according to the model I guess it does
bit weird, how would they combine semantically? as an `or` I suppose?

Reply:
Yeah I also found it strange but :woman_shrugging: